src/components/exit-this-page/index.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Exit this page
+description: Give users a way to quickly and safely exit a service, website or application.
+section: Components
+aliases:
+backlog_issue_id: 213
+layout: layout-pane.njk
+status: Experimental
+statusMessage: This component is currently experimental. <a class="govuk-link" href="#known-issues-and-gaps">You'll need to do your own research</a> to decide whether to add this component to your service.
+---
+{% from "_example.njk" import example %}
+
+Give users a way to quickly and safely exit a service, website or application.
+
+For service journeys, you must use this component with the pattern to help a user [exit a page quickly](/patterns/exit-a-page-quickly/).
+
+{{ example({group: "components", item: "exit-this-page", example: "default", html: true, nunjucks: true, open: false, size: "s"}) }}
+
+## When to use this component
+
+Use the component on pages with sensitive information that could:
+- put someone at risk of abuse or retaliation
+- reveal someone’s plans to avoid or escape from harm
+
+For example, when a potential victim is using a service to help them leave a domestic abuser.
+
+You can use this component on either:
+- all the pages in a service
+- parts of the journey with sensitive information
+
+You can also use this component for standalone content pages, such as dashboards and guidance.
+
+## When not to use this component
+
+Do not use this component if the service or content is unlikely to put a user at risk. See the [Exit a page quickly](/patterns/exit-a-page-quickly/) pattern for examples of at-risk and sensitive topics.
+
+Keep in mind that seeing this component might discourage certain users from using your service. If the user does not identify themselves as being at risk, they might see the button on a service and decide it’s not relevant to them.
+
+## How it works
+
+There are 2 ways to use the exit this page component. You can use HTML or, if you are using Nunjucks or the GOV.UK Prototype Kit, you can use the Nunjucks macro.
+
+## Adding a button
```

Comment:
Should this be a h3 under the "how it works" h2? Also a mini opinion on content.

```suggestion
### Adding the button
```

Reply:
Maybe it should be "adding the **visual** button"